src/ts/components/seekbarlabel.ts

```diff
@@ -110,6 +105,12 @@ export class SeekBarLabel extends Container<SeekBarLabelConfig> {
       this.setThumbnail(this.player.getThumbnail(absoluteSeekTarget));
     }
 
+    if (args.marker) {
+      this.setTitleText(args.marker.marker.title);
+    } else {
+      this.setTitleText(null);
+    }
```

Comment:
How about 
```suggestion
    this.setTitleText(args.marker?.marker?.title ?? null);
```
although I would recommend refactoring `setTitleText` to take an empty string to set the title to nothing, instead of a `null`

Reply:
We would need to upgrade typescript to `3.7.0` for `optional chaining`. And as the `pr` is already pretty big, that should be separate PR.

Reply:
What about the second suggestion?

Reply:
I mean sure, just trying to keep this PR as small as possible :D keep seeing stuff to refactor, it is driving me insane :D

Reply:
The reason why we always try to keep changes as small as possible as that others are forking this repo and we never know what they are changing. So to avoid merge conflicts we usually don't change stuff like that.

Reply:
Perhaps good old ternary is more compact :)
```ts
args.marker
  ? this.setTitleText(args.marker.marker.title)
  : this.setTitleText(null)
```

Reply:
Updated TS in https://github.com/bitmovin/bitmovin-player-ui/pull/312